serve.py

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/python2
+#!/usr/bin/python3
 
-import BaseHTTPServer, os, re, signal, SimpleHTTPServer, socket, sys, tarfile, tempfile, threading, time, zipfile
+import http.server, os, re, signal, http.server, socket, sys, tarfile, tempfile, threading, time, zipfile
```

Comment:
The second `http.server` was present in 1.7.1, but we may want to consider removing it...